doc/getting_started.md

```diff
@@ -0,0 +1,137 @@
+# Getting Started with Dinit
+
+In this guide we will go through the steps required to set up a tiny user-mode
+dinit instance.
+
+We assume that dinit has already been installed; i.e. we only cover
+configuration here.
+
+## Starting Dinit
+
+Let's start by running dinit with the bare minimum configuration. First we need
+to choose a place for our service descriptions to live. For this guide we are
+going to use `~/dinit`, but you can use wherever you like.
```

Comment:
There's also the possibility of using the default, `~/dinit.d`. Doesn't have to be mentioned, though.

Reply:
Let's use the default.